scripts/upload-readme.py

```
@@ -30,11 +68,11 @@ def __init__(self):
 
     def authenticate(self, api_key):
         """
-        Function to authenticate with the Readme API 
+        Function to authenticate with the Readme API
         """
         r = requests.get('https://dash.readme.com/api/v1', auth=(api_key, ''))
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)